Added styling

RVHUSA opened the pull request:
### What does this PR do?
Added styling and made a new branch

### Description of Task to be completed?
Added styling with tailwind css to the page and added a new branch for this

### How should this be manually tested?
Look over and compare to figma file

### Any background context you want to provide?
No

RVHUSA committed:
Added styling

Tomatsuppe11211 reviewed (changes requested):
you have brought the right elements. The styling need some changes here as the buttons are not over each other and most importantly right now, centered.

To do:
1) Align buttons over each other.
2) Center the buttons


Figma design:
![image](https://github.com/user-attachments/assets/9320a8e3-b2e5-4482-882f-10603e5c8727)

Your design:
![image](https://github.com/user-attachments/assets/af08e119-754b-455d-9630-29e139a6d771)


The scaling need some work (to mobile)

Figma design:
![image](https://github.com/user-attachments/assets/354d3c5e-5454-4b0a-bbcd-7c0dcedfe1fa)

Your design:
![image](https://github.com/user-attachments/assets/1fe3b38a-bc45-48a6-92ce-434cb7664a08)


For scaling, you can look at this link:
https://tailwindcss.com/docs/responsive-design

RVHUSA committed:
Merge branch 'develop' into add-students-css

RVHUSA committed:
Updated requested change in styling

Added a div for the buttons to center, and closed a section that was not closed

RVHUSA committed:
Merge branch 'add-students-css' of https://github.com/NoroffFEU/edu-gate into add-students-css

RVHUSA committed:
Update styles.css

RVHUSA committed:
Merge branch 'develop' into add-students-css

Tomatsuppe11211 commented:
Nice!
Two more things:

1) Make table wide to cover up more space (width)

2) Make the table responsive. When we scale the screen down to mobile, The table will not fit as it is too big. Smaller texts and padding should resolve this. 

Tips: in the classes, type for example: md:text-xs for making the text small for screens that is 768px or wider. 
![image](https://github.com/user-attachments/assets/578dc442-db8c-4882-a793-a68307c01179)


Check out this link to get a guide for responsive design
https://tailwindcss.com/docs/responsive-design

Figma design:
![image](https://github.com/user-attachments/assets/8adaa415-f007-4b3c-91e2-53a1a053a2f9)

your code
![image](https://github.com/user-attachments/assets/f50b200b-b77f-49f3-b1af-f393a537ba73)
